semantics.json

```diff
@@ -22,12 +22,20 @@
     "common": true
   },
   {
-    "label": "Feedback text",
+    "label": "Correct feedback text",
     "name": "feedback",
     "type": "text",
     "default": "You got @score of @total points",
-    "description": "Feedback text, variables available: @score and @total. Example: 'You got @score of @total points.'",
-    "common": true
+    "description": "Feedback text when correct answer is given, variables available: @score and @total. Example: 'You got @score of @total points.'",
+    "common": false
```

Comment:
Semantic common fields default to "false", there is no need to specify this. You can remove the "common": false line.